src/www/random.html

```diff
@@ -1,42 +0,0 @@
-<html>
```

Comment:
This file got renamed when I changed the folder name. Don't worry, it's still there.